airy.go

```diff
@@ -0,0 +1,19 @@
+package mathext
+
+import "github.com/gonum/mathext/internal/amos"
+
+// Airy returns the Airy function at z.
```

Comment:
Is it sufficient to say "The Airy function here, Ai(y), is one of the solutions to y'' - y*z = 0. See http://mathworld.wolfram.com/AiryFunctions.html" for more detailed information.